src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -275,8 +285,9 @@
               <!-- Delivery address (director/proprietor/partner only) -->
               <article v-if="isDirector || isProprietor || isPartner" class="delivery-address mt-6">
                 <v-checkbox
-                  class="mt-0 pt-0 mb-n2"
```

Comment:
Previously we did this to reduce the whitespace below the checkbox, which may contain details.

If we know this checkbox won't display details (errors), the we can use `hide-details` instead >>>